src/main/java/rosenfeld/boggle/BoggleController.java

```diff
@@ -0,0 +1,82 @@
+package rosenfeld.boggle;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.text.Text;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+
+public class BoggleController {
+    @FXML
+    Button start, submitWord;
+    @FXML
+    Text currentWord, score;
+    @FXML
+    private List<Label> letterTiles;
+    @FXML
+    TextField playerWords;
+    Game game;
+
+    private Stack <String> clickedLetters;
+    private List<String> words;
+
+    public void initialize() throws IOException {
+        clickedLetters = new Stack<>();
+        words = new ArrayList<String>();
+        game = new Game();
+    }
+
+    public void onLetterClicked(javafx.scene.input.MouseEvent mouseEvent) {
+        Label label = (Label) mouseEvent.getSource();
+        String letter = label.getText();
+
+        if (label.getStyle().equals("clicked")){
+            label.getStyleClass().remove("clicked");
+            clickedLetters.remove(letter);
+        } else {
+            label.getStyleClass().add("clicked");
+            clickedLetters.add(letter);
+        }
+        updateStack();
+    }
+
+    public void startGame() {
+        BoggleBoard boggleBoard = new BoggleBoard(4);
+        for (Label label : letterTiles) {
+            if (label.getText().isEmpty() || label.getText().equals("")) {
+                label.setText(boggleBoard.nextLetter());
+            }
+        }
+    }
+
+    public void updateStack(){
+        String word =  clickedLetters.toString()
```

Comment:
Why are you doing this?

Reply:
I'll rename it to make it clearer but it updates a label on the UI that keeps forming a word based on the letters the user is clicking (stores in a stack because I think that user can remove letters, starting with most recent)